src/code_FBE10.c

```diff
@@ -67,7 +67,27 @@ ApiStatus func_802D7B74(ScriptInstance* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "code_FBE10", func_802D7BA4);
 
-INCLUDE_ASM(s32, "code_FBE10", func_802D7CF8);
+ApiStatus func_802D7CF8(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 var0 = get_variable(script, *args++);
+    s32 var1 = get_variable(script, *args++);
+    s32 var2 = get_variable(script, *args++);
+    s32 var3 = get_variable(script, *args++);
+    s32 var4 = get_variable(script, *args++);
+    s32 var5 = get_variable(script, *args++);
+    s32 var6 = get_variable(script, *args++);
+    ScriptInstance* newScript = start_script(&D_802D9D50, 1, 0);
+
```

Comment:
Probably move the `start_script` to this line and have the empty line between the `get_variables` and it imo

Reply:
I agree with this; maybe I missed some conversation on discord, since it was marked as resolved

Reply:
Yeah, it was changed in e8ba79a88c5e2d52bc34bc3e0bff98e1ee37fc11